deployment/bicep/parameters.json

```diff
@@ -0,0 +1,25 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentParameters.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "vmazdevopsUsername": {
+            "value": "test_username"
+        },
+        "vmazdevopsPassword": {
```

Comment:
probably better to set this to something like "provide override upon deployment" and not suggest storing secrets or pwds in code